examples/call-service-panel-example/src/CallServicePanel.tsx

```diff
@@ -107,6 +107,11 @@ function CallServicePanel({ context }: { context: PanelExtensionContext }): JSX.
   );
 }
 
-export function initCallServicePanel(context: PanelExtensionContext): void {
+export const initCallServicePanel: ExtensionPanelRegistration["initPanel"] = (context) => {
```

Comment:
I find this syntax more confusing and would recommend we either export a named type for this (not pulling out the `["initPanel"]` property of another type) or just type it as `(context: PanelExtensionContext): () => void`.

Reply:
I was imagining this would be the feedback. Probably right to change this to more conventional syntax

Reply:
We all have our guilty pleasures. Mine are nested ternaries and yours are these type annotations.